hebrew/sane-option/eating_marshmelow/eating.md

```diff
@@ -0,0 +1,2 @@
+אחרי שאכלת חלק מהמרשמלו אתה מגלה שמאחורי המרשמלו באחד הקירות יש דלת סתרים, אתה דוחף אותה קלות ורואה שמבפנים בוקע אור חזק...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.